ci/test_cpp_memcheck.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# Copyright (c) 2023, NVIDIA CORPORATION.
+
+source "$(dirname "$0")/test_cpp_common.sh"
+
+EXITCODE=0
+trap "EXITCODE=1" ERR
+set +e
+
+# Run gtests with compute-sanitizer
+rapids-logger "Memcheck gtests with rmm_mode=cuda"
+export GTEST_CUDF_RMM_MODE=cuda
+COMPUTE_SANITIZER_CMD="compute-sanitizer --tool memcheck"
+for gt in "$CONDA_PREFIX"/bin/gtests/{libcudf,libcudf_kafka}/* ; do
+    test_name=$(basename ${gt})
+    if [[ "$test_name" == "ERROR_TEST" ]]; then
+        continue
+    fi
+    if [[ "$test_name" == "STREAM_IDENTIFICATION_TEST" ]]; then
+        continue
+    fi
+    echo "Running gtest $test_name"
+    echo "${COMPUTE_SANITIZER_CMD} ${gt} --gtest_output=xml:${RAPIDS_TESTS_DIR}${test_name}.xml"
```

Comment:
Maybe store the arguments (`${gt}` onward) into a variable so that you don't have to copy-paste it below? I can totally see someone needing to make some minor modification in the future and forgetting to do it on both lines.

Reply:
I'll just remove the echo line